Xamarin.Essentials/WebAuthenticator/WebAuthenticator.uwp.cs

```diff
@@ -25,6 +25,8 @@ static async Task<WebAuthenticatorResult> PlatformAuthenticateAsync(Uri url, Uri
                     // For GET requests this is a URI:
                     var resultUri = new Uri(r.ResponseData.ToString());
                     return new WebAuthenticatorResult(resultUri);
+                case WebAuthenticationStatus.UserCancel:
+                    return null;
```

Comment:
I could agree, but I'm a little torn: I know an exception is more accurate, but the `user canceled operation` is way legit, and I think **filter** an exception like `TaskCanceledException`, would be a little undue, in my opinion of course.

On the other hand, leaving the actual behavior as a `null` return, would require a full revision of the other platforms implementation, considering that they are actually throwing `TaskCanceledException` when the user cancels the flow.

If we choose to align the current UWP implementation with the other platforms, then we must update the documentation too, because it causes mess.

I just tested the current suggestion and it works fine, so the question now is: how do we want to proceed about that?

Reply:
Let's go with the exception to match the Android/iOS and update the docs.